app/models/spree/adjustment.rb

```diff
@@ -87,16 +87,6 @@ class Adjustment < ActiveRecord::Base
 
     localize_number :amount
 
-    # Update the boolean _eligible_ attribute which determines which adjustments
-    # count towards the order's adjustment_total.
-    def set_eligibility
-      result = mandatory || amount != 0
```

Comment:
what will happen with `mandatory` then? according to what you said

> Also, if an adjustment would have a zero amount when it's created, it gets ignored instead of saved.

it feels like we're not taking it into account?

Reply:
> what will happen with mandatory then?

It's still used when an adjustment is created. It will create the adjustment if it's set to mandatory and the amount is zero. See [here](https://github.com/openfoodfoundation/openfoodnetwork/blob/173cf9e536dd3c75bedc2faef1fc81c3b9cceae1/lib/spree/core/calculated_adjustments.rb#L37). The "mandatory" flag is only used on shipping and payment fee adjustments, and only when they're first created.

Reply:
This method is also removed in Spree 2.2 stable :+1: